src/App.js

```diff
@@ -0,0 +1,26 @@
+import { Routes, Route } from "react-router-dom";
+import {SignUp, Login, HomePage, ProductListing, Wishlist, Cart} from './Pages';
+import { Navbar } from "./Components/Navbar/Navbar";
+import { Footer } from './Components/Footer/Footer';
+import './App.css'
+
+function App() {
+  return (
+    <div className="App">
+      <Navbar />
+      <Routes>
+        <Route path="/" element={<HomePage />} />
+        <Route path="/HomePage" element={<HomePage />} />
```

Comment:
Already defined `/` above for `Homepage` then why this route again?

Reply:
I added the extra link because, when i clicked on homepage Link. It was redirecting me to some other blank page instead of the Homepage.

Reply:
For this debug the issue but adding two routes for the same destination is not good imo.